src/foam.js

```diff
@@ -144,13 +149,14 @@
 
       m.code();
     },
+    poms: {},
```

Comment:
When this is an object we have less control over ordering. If it was a list of objects, where each object has `foam.cwd` and `pom` in two separate properties, journals will always be loaded in the same order

Reply:
`{}` is a hashmap rather than an associative array - but, by coincidence the three projects currently there (`foam/src`, `foam/src/nanos`, `foam/src/support`) are iterated in an order that makes sense.